### PR TITLE
Hardcoded pointers

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: compare.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# create baserom.txt if necessary
+
+if [ ! -f baserom.txt ]; then
+	hexdump -C baserom.gbc > baserom.txt
+fi
+
+hexdump -C pokepinball.gbc > pokepinball.txt
+
+diff -u baserom.txt pokepinball.txt | less
```
Comment (dannye) [inline compare.sh:12]: Again with this compare.sh? Why do you feel this is necessary?

**Comment by huderlem** [comment]
I'm glad this project is getting worked on again 👍

File: main.asm [hunk 2]
```diff
@@ -21261,8 +21261,8 @@ Func_106b6: ; 0x106b6
 	rl b  ; bc was just multiplied by 16
 	ld hl, PokemonNames + 1
 	add hl, bc
-	ld de, $2a67 ; todo "You got an"
-	ld bc, $2a91 ; todo
+	ld de, YouGotAnText ; todo "You got an"
```
Comment (huderlem) [inline main.asm]: You can remove the bunch of "todo" comments.

File: compare.sh (hunk 1 above)
Comment (PikalaxALT) [reply compare.sh:12]: It was extremely useful while disassembling Crystal; whenever something didn't match, this script would point me to where the mismatch is.

**Comment by huderlem** [comment]
Is this ready to merge?  It's just been sitting for awhile.  I'm going to be starting up work again on pokepinball, and it would be cool if these changes were in master.

**Comment by huderlem** [comment]
This PR is missing a data/oam_frames.asm file.  However, I've added it locally and confirmed a clean build.  I will merge this PR now, and push the missing file to the master branch.  Thanks, Pikalax!

**Pull request merged**